apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -104,7 +121,7 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
           await deleteThread({
             threadId: params?.threadId ?? activeThread?.threadId,
             jwt: session!.user?.hasuraJwt,
-            userId: session!.user.id,
+            userId: session!.user.id
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)